queue_job_chunk/models/queue_job_chunk.py

```diff
@@ -0,0 +1,55 @@
+
+from odoo import models, fields
+from odoo.addons.queue_job.job import job
+
+
+class QueueJobChunk(models.Model):
+    _name = "queue.job.chunk"
+
+    def _compute_reference(self):
+        for rec in self:
+            model = rec.model_id
+            record = rec.record_id
+            if model and record:
+                rec.reference = self.env[model].browse(record)
+
+    # component fields
```

Comment:
We should add a company field.
The company field should be the company_id of the object linked by the reference field (if a company_id field exist) or the company of the user (if not field company on the related reference object)